cmd/broker/filter/main.go

```diff
@@ -76,10 +78,18 @@ func main() {
 		log.Fatal("Error building kubeconfig", err)
 	}
 
+	var env envConfig
+	if err := envconfig.Process("", &env); err != nil {
+		log.Fatal("Failed to process env var", zap.Error(err))
+	}
+
 	ctx, _ = injection.Default.SetupInformers(ctx, cfg)
 	kubeClient := kubeclient.Get(ctx)
 
-	loggingConfig, err := sharedmain.GetLoggingConfig(ctx)
+	loggingConfigMapName := cmpresources.MakeCopyConfigMapName(namespaceresources.DefaultConfigMapPropagationName, logging.ConfigMapName())
```

Comment:
Shouldn't we have a fallback in case ConfigMapPropagation is disabled (for any reason)?

Reply:
Could you provide some cases when ConfigMapPropagation will be disabled? And I was wondering what kind of fallback you think we should have (for making the broker reconcile successfully or just for that loggingConfigMapName and metricConfigMapName?)

Reply:
I didn't have any particular scenario for disabling replication in mind, that was a genuine open question because I didn't follow early conversations on that topic. The fallback I was thinking about was to the original hardcoded "kn-eventing/config-xyz" Configmaps, but if config replication becomes a core Kn feature which guarantees a Broker can always expect a copy of the logging and observability configmaps to be present, I'm fine with the implementation :+1:

Reply:
Thanks! Before this CRD ConfigMapPropagation was introduced, what we used to do is letting broker get the data from original hardcoded ConfigMaps which were in "knative-eventing" namespace (with some RBAC rule). Currently, ConfigMapPropagation only used when a namespace is injecting label "knative-eventing" to start a broker. If the ConfigMapPropagation is disabled, this part of the code in namespace will be modified, and we can set back RBAC rules for the broker and corresponding broker reading ConfigMap logic to use the old way (which would be a fallback as you suggested).